<launch>

  <arg name="rviz" default="true" />

  <!-- <node pkg="fast_lio" type="imu_process" name="imu_process" output="screen"/>


  <node pkg="fast_lio" type="loam_scanRegistration_horizon" name="scanRegistration_horizon" output="screen">
  </node>


  <node pkg="fast_lio" type="loam_laserMapping" name="laserMapping" output="screen">
  <param name="map_file_path" type="string" value=" " />
  <param name="filter_parameter_corner" type="double" value="0.3" />
  <param name="filter_parameter_surf" type="double" value="0.2" />
  </node>

  <node pkg="fast_lio" type="livox_repub" name="livox_repub" output="screen" >
  <remap from="/livox/lidar" to="/livox/lidar" />
  </node> -->

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio)/rviz_cfg/loam_livox.rviz" />
  </group>

</launch>